### ISSUE TITLE
Slowness after screen lock

### ISSUE THREAD

**Issue by wifiuk** [rsmudge/armitage#189]
I have noticed if my machine goes to screen lock , when i log back in again, armitage takes ages to run any command or exploit.

I exit the app, load it back up again and all back to normal.

**Comment by TheShadow88** [comment]
Yes, armitage can become slow after a ton of different situations .

For me it often happens after compromising a non-windows host, it's weird

**Comment by TheShadow88** [comment]
I guess the only way to fix it, is just to have metasploit running with an rpc server on and when the slowness happens to close it and reconnect it to the RPC
